### ISSUE TITLE
I would like some binaries, please, if possible

### ISSUE THREAD

**Issue by unforgettableid** [phhusson/Superuser#1]
Dear @phhusson, @jeekajoo, @AndyLiu008, and others:

Hi!

I wonder if someone could please build binaries of phhusson/Superuser and upload them somewhere.  Maybe to your website or Dropbox or Google Drive or somewhere.

You will need to use https://github.com/phhusson/Widgets/ in order to complete the build.

There's a [Docker] image in @jeekajoo's GitHub space [here](https://github.com/jeekajoo/docker-android) which might help you.

Thanks!

**Comment by xfausto** [comment]
What is the state of getting a flashable zip now? After Chainfire announced that SuperSU was bought by a company, people are looking for an alternative.

**Comment by mydongistiny** [comment]
Here's a flashable zip built with the source available today. http://files.oceighty.co/other/Superuser-09292015.zip

**Comment by xfausto** [comment]
I saw your comment on my e-mail. I'll try it on the AOSP rom of my phone and I'll provide feedback about it.

**Comment by mydongistiny** [comment]
The apk doesn't seem to parse. Everything else works. I have to look into the apk again tomorrow or something.

**Comment by phhusson** [comment]
Your zip doesn't copy placeholder, doesn't rename app_process32, doesn't set any SELinux context
There is no way it works...

**Comment by mydongistiny** [comment]
Yeah. I kept getting problems with ant erroring out so I built using ndk and just tried to copy the zip files bases off of an old build.xml. ;-)

**Comment by xfausto** [comment]
It seems that people are getting interested in this project. How could they help to improve this fork?

**Comment by cubakcabak** [comment]
currently using khous latest playstore superuser version on kitkat with selinux enforced, manually installing to the system thorough recovery adb, because the installations script failed.

so far its running without hiccups, but i dont know the effect of lacking selinux support on it.

i wish this project ready to release usable binary. because supersu is "never work" on this system, some chinese branded phone.

**Comment by cubakcabak** [comment]
so, i came a cross a glitch, finally.

titaniumbackup doesnt "work right", i guess thats one side effect of the old superuser without proper selinux support.

still hopping you releasing this superuser soon enough.

**Comment by unforgettableid** [comment]
@cubakcabak:  Please file a new issue instead of discussing the matter here in this thread.  And, when you do so, please provide lots of details.  Try to write at least 100 words.

**Comment by kousu** [comment]
Is there an update on the state of .zip files? Is see that https://github.com/phhusson/Superuser/blob/master/Superuser/build-zip.sh exists, but it would be a lot easier to use if the output of that was around too: you could just host it in the repo.
Chainfire has a single https://download.chainfire.eu/896/SuperSU/BETA-SuperSU-v2.66-20160103015024.zip?retrieve_file=1 which contains subfolders for every architecture, so this is possible, though presumably tedious.

Thanks for the fork!

**Comment by 0nse** [comment]
@kousu http://superuser.phh.me/superuser.zip ?

**Comment by kousu** [comment]
Oh, great! But this will wipe my boot partition, won't it? I didn't notice the link on the README because it says "I setup a robot to build root-ed boot.img." so I was expecting to find a boot.img that I needed to `fastboot flash`, and that sounds scary, and like I need to have done some work (I tend to buy cheap phones which tend not to have ROMs floating around).

ChainFire's edits /system instead and works universally.  Am I fearful over nothing?

**Comment by 0nse** [comment]
This is a flashable ZIP and not a fastboot partition image. The boot partition is not wiped afair. Thus, you need a custom recovery.

**Comment by kousu** [comment]
That's what I am used to using, which is great. But the comments in the README make me wary, and http://superuser.phh.me/superuser.zip:/META-INF/com/google/android/update-binary says
```
ui_print "Found bootimg @ $bootimg"
sh -x ../bootimg.sh $bootimg $modes
ui_print "Generated $pwd/new-boot.img"
dd if=new-boot.img of=$bootimg bs=8192
ui_print "Flashed root-ed boot.img"
```
:fearful: 
Oh but bootimg.sh looks like it's extracting, editing, and repacking the boot partition, not just summarily overwriting it. But I'm not totally sure.

@phhusson , should this .zip work on *any* Android or do I need to extract my ROM and send it in a PR first?

**Comment by kousu** [comment]
Also the [description on F-Droid](https://f-droid.org/repository/browse/?fdfilter=superuser&fdid=me.phh.superuser) says "if you know what you are doing you can use this apk as a su/Superuser substitute if you are gaining an initial root." which sounds a lot like you have to unzip the .apk and hack it up in some undocumented way that you must have learned by lurking XDA.

If it really is simple enough as that this zip is universal, please update the description on the README to link to it directly and make that clear! And try to get the F-Droid description updated with a link to that part of the README :)

**Comment by phhusson** [comment]
If your device is not stupid and you can always boot to recovery, even if
boot.img is corrupted and you have a backup, then you can just try.
But yes the aim is to be as universal as possible.
